### PR TITLE
Move documentation views and assets from docs folder into app folder

### PR DESCRIPTION
**Pull request by frankieroberto**
This takes all the existing guidance and examples in docs/ and moves it to app/.

Main effect of this is that everything at `/docs/` in the URL path moves to the root (`/`).

Also simplifies things a bit by not having 2 routes files, assets folders, etc.

### CODE REVIEW

**Comment by frankieroberto** [comment]
@edwardhorsford @anandamaryon1 @vickytnz no hurry but if anyone could help test this by running the branch locally that'd be 🙌!

File: app.js [hunk 1]
```diff
@@ -204,9 +203,6 @@ if (useDocumentation || onlyDocumentation === 'true') {
     extended: true,
   }));
```
Comment (edwardhorsford) [inline app.js:204]: Delete this whole section?

File: app.js [hunk 2]
```diff
@@ -204,9 +203,6 @@ if (useDocumentation || onlyDocumentation === 'true') {
...
 
-  // Custom documentation routes
-  documentationApp.use('/', documentationRoutes);
-
   // Automatically route documentation pages
```
Comment (edwardhorsford) [inline app.js:206]: Delete?

Comment (frankieroberto) [reply app.js:206]: Wasn't sure if this bit was still needed?

File: app.js (hunk 1 above)
Comment (frankieroberto) [reply app.js:204]: Let's do it in a follow-up PR, removing the password stuff

**Review by edwardhorsford — APPROVED** [review]
Have run locally and slicked around a bit. Seems to work!

There's still a bunch of docs references to remove, but as discussed, those can be done in a later PR.

**Pull request merged**